python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -880,6 +883,25 @@ keys:
                       max: 65535
                       convert_types:
                         - str
+            aaa:
+              type: dict
+              keys:
+                unresponsive:
+                  type: dict
+                  description: Configure AAA timeout options.
+                  keys:
+                    eap_response:
+                      type: str
+                      description: EAP response to send.
+                      valid_values:
+                        - "success"
+                        - "disabled"
+                    action:
+                      type: dict
+                      $ref: "eos_cli_config_gen#/keys/dot1x/keys/aaa/keys/unresponsive/keys/action"
```

Comment:
the configs in interface and global dot1x are not same (having access_list in interface),
not sure if its for specific version, please recheck.

```
s1-leaf1(config-dot1x)#aaa unresponsive action traffic allow ?
  vlan  Allow traffic in VLAN when AAA times out
  <cr>  
```


```
s1-leaf2(config-if-Et2)#dot1x aaa unresponsive action traffic allow ?
  access-list  Apply access-list when AAA times out
  vlan         Allow traffic in VLAN when AAA times out
  <cr>  
```

Reply:
Added access-list support.

Reply:
https://github.com/aristanetworks/avd/pull/4191#discussion_r1682265067